app/views/admin/unattached_promo_registrations/index.html.slim

```diff
@@ -9,6 +9,9 @@
         hr
         .
           = number_field_tag "number_of_codes_to_create", nil, placeholder: "# of codes"
+          br
+          = check_box_tag "hidden"
+          = label_tag "Hide from publishers (Used for testing, checking this is uncommon)"
```

Comment:
Let's make sure the checkbox is in the label tag so that you can click the text for the label and it'll check the box